CI/before_install/linux_qt5.sh

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env bash
 
-set -euo pipefail
+set -euxo pipefail
 export DEBIAN_FRONTEND=noninteractive
 
+# https://github.com/microsoft/onnxruntime/discussions/6489
+ONNXRUNTIME_URL=https://github.com/microsoft/onnxruntime/releases/download/v1.18.1/onnxruntime-linux-x64-1.18.1.tgz
+ONNXRUNTIME_ROOT=/opt/onnxruntime
+sudo mkdir -p "$ONNXRUNTIME_ROOT"
+curl -fsSL "$ONNXRUNTIME_URL" | sudo tar -xzv --strip-components=1 -C "$ONNXRUNTIME_ROOT"
+find "$ONNXRUNTIME_ROOT"  # print file tree
```

Comment:
maybe extract this part to a shared file? also this comment is different in the two files